### ISSUE TITLE
Base clean up

### ISSUE THREAD

**Issue by johnsmm** [Liveandletdie/becti_zerty_OFPS_Extended.Altis#26]
I need some one to take the script from CRcti that cleans up the base.
It removes broken buildings trees etc.

**Comment by apollodeathstar** [comment]
what version of the CRCTI? the proman public version? crCTI Kastenbier Edition? can you give link to specific one you would like to be pulling from? I'll try to pull pull it :S

**Comment by Liveandletdie** [comment]
Dont add the removal of trees part if you tackle this, the cover is nice for some base locations!

**Comment by apollodeathstar** [comment]
Is it possible to adapt this for a "Clear runway" script? so any flat area the commander can pay like $100000 to clear out all objects in a large strip thus making allot of areas usefull?

**Comment by Liveandletdie** [comment]
I do like that idea....especially if they could remove the rock walls! That might require a bit of work but if we can figure it out that would be a very nice addition

**Comment by apollodeathstar** [comment]
Yeah thats what I was thinking, If it can remove rock walls and the rocks on map, would be great, if it can clear buildings and poles and stuff from beside straight stretches of hwy, would be amazing!

**Comment by johnsmm** [comment]
I think apollo means the fucking crash rocks that noobs leave when they crash.

What id like the script to do is clean up air wrecks "rocks" and broken buildings, roofs, hangers around base in 1000 meter radius.

and its CRcti Kastenbier Edition
